server/handler/issue_comment.go

```diff
@@ -46,6 +47,12 @@ func (h *IssueComment) Handle(ctx context.Context, eventType, deliveryID string,
 	installationID := githubapp.GetInstallationIDFromEvent(&event)
 	ctx, logger := githubapp.PreparePRContext(ctx, installationID, repo, number)
 
+	// Debouncing debug comments from bot to prevent an infinite loop
+	if event.GetComment().GetUser().GetLogin() == fmt.Sprintf("%s[bot]", h.AppName) {
```

Comment:
Hm, it appears that the event contains the `installation ID`  (which we already use on L46) and doesn't include the app's `integration ID`. I agree if we had that it would be easier to use instead of using the app client on server startup.